src/admin/class-cmb2.php

```diff
@@ -147,32 +169,49 @@ public static function cmb2_render_openstreetmap_field_type( $field, $escaped_va
 				}
 
 				$geometry_array = json_decode( $geometry_object[0], true );
+
+				// Check if the geometry object has a type Feature wrapper or not.
+				if ( 'Feature' !== $geometry_array['type'] ) {
+					$geometry_array = [
+						'type'       => 'Feature',
+						'geometry'   => $geometry_array,
+						'propreties' => [],
+					];
+				}
+
 				if ( empty( $geometry_array['geometry']['coordinates'] ) ) {
```

Comment:
Een geometry kan ook bestaan uit een array van geometries, dan faalt deze check nu, terwijl het wel valide is.

Dus het kan ook zijn: `$geometry_array['geometries'][0]['coordinates']`

Reply:
@richardkorthuis : heb je daar een voorbeeld bestand van toevallig?

Reply:
@eyalacato Niet direct een voorbeeldbestand, wel een voorbeeld in de specificatie: https://stevage.github.io/geojson-spec/#appendix-A.7